### DISCUSSION TITLE
MIDI Input in Neovim!

### DISCUSSION THREAD

**Discussion by niveK77pur** [martineausimon/nvim-lilypond-suite#28] [Show and tell]
Hello! I wanted to share my neovim plugin here as well after posting it [on reddit](https://www.reddit.com/r/neovim/comments/17udvh0/async_and_modal_midi_input_in_neovim_my_first/). (A bit of a shameless plug if you allow me lol)

<https://github.com/niveK77pur/midi-input.nvim>

It effectively brings MIDI input right into Neovim (similar to what you find in Frescobaldi), please check out the plugin's README which provides many more details!

Your plugin proves extremely useful and provides a lot of handy features for Lilypond development in neovim, so I thought it could be interesting to mention my new plugin in your tips and tricks section as well! *Only if you want to of course!* :D

**Comment by martineausimon** [comment]
Hi Kevin, thanks for your message, and congratulations on this plugin, it's super useful ! I can't test it right now because I don't have a MIDI device, but I'm looking forward to seeing how it works.
Anyway, the code is really well written, I should take inspiration from it, especially `job.lua`, to improve nvim-lilypond-suite ! Of course, I'll add this to the tips and tricks section ;-)

**Reply by niveK77pur** [reply]
Awesome happy to hear and thanks! Should you have any questions regarding my implementation/code, I will of course be more than happy to help!
